src/utils/debounce.spec.ts

```diff
@@ -0,0 +1,91 @@
+import {debounce} from './debounce';
+
+describe('debounce', () => {
```

Comment:
is it possible to provide a test showing the main purpose of debounce? `func` is called several times, but is only called once during at the end of debounce time

Reply:
added multiple fn() calls